frontend/src/lobby/GameOptions.jsx

```diff
@@ -54,6 +57,30 @@ const Sets = ({ sets, type }) => (
     .map((set, i) => <Set type={type} selectedSet={set} index={i} key={i} />)
 );
 
+const Decadent = ({ sets, type }) => (
+  <Fragment>
+    <blockquote className="game-mode-description">
```

Comment:
I love it, could you also add some description for the other modes?

Reply:
Thought the same already - it's awesome! 😍 
But that can also be a new PR I guess...

Reply:
Glad you like it! Done. https://github.com/dr4fters/dr4ft/pull/990/commits/5850b20df108e249facc6d4d1d19f1ad789e196d

Could use some help wordsmithing.

Reply:
Wording seems ok for a first version. 👍 
Don't want to bloat this PR even more with a bunch of suggestions regarding that.